lib/unittest.bzl

```diff
@@ -59,6 +68,19 @@ unittest_toolchain = rule(
     },
 )
 
+def _impl_function_name(impl):
+    """Derives the name of the given rule implementation function.
+
+    This can be used for better test feedback."""
```

Comment:
Normally the closing '"""' goes on the next line.

Reply:
Done